src/Transformations.jl

```diff
@@ -222,7 +222,7 @@ end
 RescaledZScore(scale) = RescaledZScore(scale, nothing)
 
 compute_normalization(r::RescaledZScore, transformation, fts) =
-    RescaledZScore(r.scale, compute_normalization(r.zscore, transformation, fts))
```

Comment:
For a new `RescaledZScore` object, `r.zscore == nothing` causing `compute_normalization` to return `nothing`, leading to the data being untouched in line 228 where it is supposed to be normalized.

Reply:
Nice catch. Maybe also safer to use the constructor

```julia
RescaledZScore(scale) = RescaledZScore(scale, ZScore())
```

Reply:
Sure